SonicWall/sonicwall-fw/ingest/parser.yml

```diff
@@ -0,0 +1,355 @@
+          44: "ipv6-frag"
+          45: "idrp"
+          46: "rsvp"
+          47: "gre"
+          48: "dsr"
+          49: "bna"
+          50: "esp"
+          51: "ah"
+          52: "i-nlsp"
+          53: "swipe"
+          54: "narp"
+          55: "mobile"
+          56: "tlsp"
+          57: "skip"
+          58: "ipv6-icmp"
+          59: "ipv6-nonxt"
+          60: "ipv6-opts"
+          61: "any host internal protocol"
+          62: "cftp"
+          63: "any local network"
+          64: "sat-expak"
+          65: "kryptolan"
+          66: "rvd"
+          67: "ippc"
+          68: "any distributed file system"
+          69: "sat-mon"
+          70: "visa"
+          71: "ipcv"
+          72: "cpnx"
+          73: "cphb"
+          74: "wsn"
+          75: "pvp"
+          76: "br-sat-mon"
+          77: "sun-nd"
+          78: "wb-mon"
+          79: "wb-expak"
+          80: "iso-ip"
+          81: "vmtp"
+          82: "secure-vmtp"
+          83: "vines"
+          84: "ttp"
+          85: "nsfnet-igp"
+          86: "dgp"
+          87: "tcf"
+          88: "eigrp"
+          89: "ospfigp"
+          90: "sprite-rpc"
+          91: "larp"
+          92: "mtp"
+          93: "ax.25"
+          94: "ipip"
+          95: "micp"
+          96: "scc-sp"
+          97: "etherip"
+          98: "encap"
+          99: "any private encryption scheme"
+          100: "gmtp"
+          101: "ifmp"
+          102: "pnni"
+          103: "pim"
+          104: "aris"
+          105: "scps"
+          106: "qnx"
+          107: "a/n"
+          108: "ipcomp"
+          109: "snp"
+          110: "compaq-peer"
+          111: "ipx-in-ip"
+          112: "vrrp"
+          113: "pgm"
+          114: "any 0-hop protocol"
+          115: "l2tp"
+          116: "ddx"
+          117: "iatp"
+          118: "stp"
+          119: "srp"
+          120: "uti"
+          121: "smp"
+          122: "sm"
+          123: "ptp"
+          124: "isis over ipv4"
+          125: "fire"
+          126: "crtp"
+          127: "crudp"
+          128: "sscopmce"
+          129: "iplt"
+          130: "sps"
+          131: "pipe"
+          132: "sctp"
+          133: "fc"
+          134: "rsvp-e2e-ignore"
+          135: "mobility header"
+          136: "udplite"
+          137: "mpls-in-ip"
+          138: "manet"
+          139: "hip"
+          140: "shim6"
+          141: "wesp"
+          142: "rohc"
+          253: "unassigned"
+          254: "use for experimentation and testing"
+          255: "reserved"
+        mapping:
+          parsed_event.message.proto: network.transport
+
+      - translate:
+        dictionary:
+          0: 'base event'
+          1: 'aggregated'
+          2: 'correlation'
+          3: 'action'
+        mapping:
+          parsed_event.message.type: cef.event_type
+        fallback: 'base event'
+
+  set_cef_extension_fields_cef:
+    actions:
+      - set:
+          cef: '{{parsed_event.message}}'
```

Comment:
As only `cef.type` is declared in your taxonomy, you can remove this part, as the result will be always empty.
```suggestion
```